Example/Backpack/Chips.storyboard

```diff
@@ -40,6 +40,7 @@
                             <constraint firstItem="h4y-YC-Uzb" firstAttribute="trailing" secondItem="HBs-ET-Q43" secondAttribute="trailing" id="4oR-S8-fnE"/>
                             <constraint firstItem="HBs-ET-Q43" firstAttribute="leading" secondItem="h4y-YC-Uzb" secondAttribute="leading" id="CT7-Sb-uGK"/>
                             <constraint firstItem="HBs-ET-Q43" firstAttribute="centerX" secondItem="Kec-l1-3BA" secondAttribute="centerX" id="LxL-12-tVe"/>
+                            <constraint firstItem="HBs-ET-Q43" firstAttribute="top" relation="greaterThanOrEqual" secondItem="h4y-YC-Uzb" secondAttribute="top" id="ocY-fI-YTd"/>
```

Comment:
Had to add this constraint as NavBars work slightly differently in iOS 13.2, resulting in this:
<img width="559" alt="image" src="https://user-images.githubusercontent.com/30267516/69315342-34acf300-0c2e-11ea-8e29-fabbf43198eb.png">

The `Belgium` chip cannot be clicked due to being behing the title!